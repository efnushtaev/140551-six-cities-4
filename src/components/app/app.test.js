import React from 'react';
import renderer from 'react-test-renderer';
import Main from './../main/main';
import {describe, expect, it} from '@jest/globals';
import {MemoryRouter} from 'react-router-dom';

describe(`appTest`, () => {
  it(`App should be rendered correct`, () => {
    const tree = renderer
      .create(
          <MemoryRouter
            initialEntries={[{pathname: `/`}]}
          >
            <Main/>
          </MemoryRouter>)
      .toJSON();

    expect(tree).toMatchSnapshot();

  });
});
