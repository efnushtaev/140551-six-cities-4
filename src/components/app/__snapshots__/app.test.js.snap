// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`appTest App should be rendered correct 1`] = `
<div>
  <button
    className="title"
    onClick={[MockFunction]}
  >
    Offers count is 
    3
  </button>
  <p>
    first-name
  </p>
  <p>
    second-name
  </p>
</div>
`;
